src/Forms/Prism.Forms/Modularity/ModuleInfo.cs

```diff
@@ -149,6 +152,6 @@ string IModuleInfo.Ref
         /// <summary>
         /// Gets or sets the state of the <see cref="ModuleInfo"/> with regards to the module loading and initialization process.
         /// </summary>
-        ModuleState IModuleInfo.State { get; set; }
+        public ModuleState State { get; set; }
```

Comment:
I've had to change the visibility of that field in order to get the bindings for the samples to work properly. It seems that the Forms bindings engine is allowing the use of non-public members, something the UWP engine does not allow.

Is there a reason to keep this property, and `Ref` above as explicit implementations ?

Reply:
we don't want people to even see the Ref property since it isn't used. The State property was actually a bug as well for Forms... this should be done a little different though since we don't want people thinking this is a property they should be setting if they manually construct the ModuleInfo.

```cs
public ModuleState State { get; private set; }
ModuleState IModuleInfo.State
{
    get => State;
    set => State = value;
}
```

Reply:
Excellent, will change that.